### ISSUE TITLE
Regarding test datset

### ISSUE THREAD

**Issue by ananditajam** [ZJLAB-AMMI/Restormer-Plus#2]
I used your pretrained weights (deraining.pth) to test the Restormer base model on test.py file and achieved a PSNR of 22. However, when I trained the model with pretrained weights with the training code and used the checkpoints in the testing code to resume from an epoch, I achieved a PSNR of 23. Why is the PSNR lower compared to your reported PSNR of 25?

**Comment by codingbus821** [comment]
> I used your pretrained weights (deraining.pth) to test the Restormer base model on test.py file and achieved a PSNR of 22. However, when I trained the model with pretrained weights with the training code and used the checkpoints in the testing code to resume from an epoch, I achieved a PSNR of 23. Why is the PSNR lower compared to your reported PSNR of 25?

My understanding is that the reported PSNR of 25 might have been evaluated on the challenge’s hidden test set, which was not publicly released. Therefore, reproducing the exact score using the available test data could be difficult.
